src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
Well as I can see this is also the case for all other scripts, if timer == diff the timer gets stuck.